### DISCUSSION TITLE
Reccuring payment - payment modules for payment with cards

### DISCUSSION THREAD

**Discussion by DegrizNet** [OpenMage/magento-lts#4150] [Extensions & Modules]
Hello,

are there some modules available that work with recurring payment on OpenMage besides PayPal? 

I know it was possible couple of years ago but I do not remember what I used.

Thank you for any info!

**Comment by sreichel** [comment]
Mageworx and aheadworks have something like that.

**Reply by DegrizNet** [reply]
https://aheadworks.com/subscriptions-and-recurring-payments-extension-for-magento-1
I have found this but I don't think it still works as it was last updated in 2017. :)

Thank you for replay!!

**Reply by sreichel** [reply]
They offfer 30 days to refund. I think there can be issues for php8, but that should be easy to fix.

**Reply by DegrizNet** [reply]
I don't know.. base on info (limited payment options and unknown support) I think it would be better for me to invest in new payment module or migration to M2.

**Comment by justinbeaty** [comment]
I think the old Gene Braintree module supports recurring payments, but I’ve never actually used them. I have a fork of the module for OpenMage here: https://github.com/justinbeaty/module-gene-braintree

**Reply by DegrizNet** [reply]
Thanky for feedback and module! I have checked and recurring payment via module is not supported.
